src/test/scala/com/tsunderebug/scolortesting/OpenTypeFontTest.scala

```diff
@@ -0,0 +1,38 @@
+package com.tsunderebug.scolortesting
+
+import com.tsunderebug.scolor.otf.OpenTypeFont
+import com.tsunderebug.scolor.otf.tables.OpenTypeNAMETable
+import com.tsunderebug.scolor.otf.types.{NameRecord, OTFString}
+import com.tsunderebug.scolor.otf.types.gen.WindowsLanguage
+
+import org.scalatest.FlatSpec
+
+class OpenTypeFontTest extends FlatSpec {
+
+  val expected = """|
+    |4f 54 54 4f 00 01 00 10 00 01 00 00 6e 61 6d 65
+    |f2 cb aa fa 00 00 00 1c 00 00 00 4c 00 00 00 00
+    |00 00 00 03 00 2a 00 03 00 01 04 09 00 04 00 0b
+    |00 75 00 03 00 01 04 13 00 04 00 0c 00 80 00 03
+    |00 01 04 1d 00 04 00 0b 00 8c 48 65 6c 6c 6f 20
+    |57 6f 72 6c 64 48 61 6c 6c 6f 20 57 65 72 65 6c
+    |64 48 65 6a 20 56 c3 a4 72 6c 64 65 6e""".stripMargin.trim
```

Comment:
I think I handled my strings wrong in the sense that they should be UTF16 but I'll fix this myself.

Reply:
I have confirmed that the strings have to be in UTF-16BE. Update this string to

```
4f 54 54 4f 00 01 00 10 00 01 00 00 6e 61 6d 65
07 bc 13 0c 00 00 00 1c 00 00 00 4c 00 00 00 00
00 00 00 03 00 2a 00 03 00 01 04 09 00 04 00 0b
00 75 00 03 00 01 04 13 00 04 00 0c 00 80 00 03
00 01 04 1d 00 04 00 0b 00 8c 00 48 00 65 00 6c
00 6c 00 6f 00 20 00 57 00 6f 00 72 00 6c 00 64
00 48 00 61 00 6c 00 6c 00 6f 00 20 00 57 00 65
00 72 00 65 00 6c 00 64 00 48 00 65 00 6a 00 20
00 56 00 e4 00 72 00 6c 00 64 00 65 00 6e
```

Reply:
I've updated the test to use this and it is now failing (as expected?)

```
[info] - should create the expected bytes *** FAILED ***
[info]   Expected "...1 00 00 6e 61 6d 65
[info]   [07 bc 13 0c 00 00 00 1c 00 00 00 4c 00 00 00 00
[info]   00 00 00 03 00 2a 00 03 00 01 04 09 00 04 00 0b
[info]   00 75 00 03 00 01 04 13 00 04 00 0c 00 80 00 03
[info]   00 01 04 1d 00 04 00 0b 00 8c 00 48 00 65 00 6c
[info]   00 6c 00 6f 00 20 00 57 00 6f 00 72 00 6c 00 64
[info]   00 48 00 61 00 6c 00 6c 00 6f 00 20 00 57 00 65
[info]   00 72 00 65 00 6c 00 64 00 48 00 65 00 6a 00 20
[info]   00 56 00 e4 00 72 00 6c 00 64 00 65 00] 6e", but got "...1 00 00 6e 61 6d 65
[info]   [f2 cb aa fa 00 00 00 1c 00 00 00 4c 00 00 00 00
[info]   00 00 00 03 00 2a 00 03 00 01 04 09 00 04 00 0b
[info]   00 75 00 03 00 01 04 13 00 04 00 0c 00 80 00 03
[info]   00 01 04 1d 00 04 00 0b 00 8c 48 65 6c 6c 6f 20
[info]   57 6f 72 6c 64 48 61 6c 6c 6f 20 57 65 72 65 6c
[info]   64 48 65 6a 20 56 c3 a4 72 6c 64 65] 6e" (OpenTypeFontTest.scala:37)
```